jetty-server/src/test/java/org/eclipse/jetty/server/ServerConnectorTimeoutTest.java

```diff
@@ -134,7 +135,7 @@ private String getResponse(String request) throws IOException, InterruptedExcept
     @Test
     public void testHttpWriteIdleTimeout() throws Exception
     {
-        _httpConfiguration.setIdleTimeout(500);
+        _httpConfiguration.setBlockingTimeout(500);
```

Comment:
@lachlan-roberts why did you revert this to be a blocking timeout ? Your change (using `setIdleTimeout()`) was correct.

Reply:
@sbordet 
you said this on the [issue](https://github.com/eclipse/jetty.project/issues/2424) that `"the test is testing the blocking timeout, so can't be replaced with an idle timeout - although it does fix the ReadPendingException."`

Reply:
@lachlan-roberts that was `ServerTimeoutsTest`, this is `ServerConnectorTimeoutTest`.
Plus the method name really tells that's an idle timeout we want to test.

Reply:
i guess you thought i was talking about `ServerTimeoutsTest.testBlockingTimeoutWithSlowRead` which was the issue title but was referring to the error log Joakime posted which was this test.

Reply:
ill change it back